### ISSUE TITLE
Feature Request: Change "Ready to build?" prompt behavior when queueing build of multiple projects

### ISSUE THREAD

**Issue by LB-ArturZieba** [o3de/o3de#10522]
**Is your feature request related to a problem? Please describe.**
Currently, if there are multiple projects queued for build in the Project Manager, each one of them gets the "Ready to build?" prompt at the start. This blocks them from automatically going through, and requires user attention. While this is not a problem when for example building two projects simultaneously on the Pre-built SDK engine configuration, on Source this could cause an unaware user to lose hours of progress.

**Describe the solution you'd like**
Either remove the "Ready to build?" prompts on subsequent queued project builds in the Project Manager, or display the prompt right away when starting the build on additional projects.

**Additional context**

https://user-images.githubusercontent.com/86952082/177304120-a942a73f-fbc8-4339-bac6-f9195108e9ff.mp4

**Comment by amzn-changml** [comment]
Adding SIG-UIUX as this seems to be a project manager workflow request

**Comment by yuyihsu** [comment]
Assigned to @amzn-leenguy . Lee will provide a solution for this issue for the dev to fix.

**Comment by amzn-leenguy** [comment]
My recommendation here for ideal UX is:
If a user was to start a new action to build, while another project is already building, we should start a queue – and make the new project go into that queue.

At 0:20s of your video, I disagree that when clicking build on another project, it stops the current one building starts the one that you acted on. 

What are your thoughts on my recommendation?

**Comment by LB-ArturZieba** [comment]
I agree that the proposed Epic/Steam-like queue system would be a better solution than the ones I've listed.

**Comment by amzn-leenguy** [comment]
UX has added this to our roadmap, and will be looking into making some actionable designs/solutions for this issue. @bhanuja-s has been assigned to this ticket to action.
